doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,280 @@
+  additional variable attributes that can be defined here (see the already
+  available cmorizers). 
+
+It is recommended to use an existing custom table as a template, to edit the content and save it as
+``<short_name>.dat``.
+
+2. Edit your configuration file
+=====================
+
+Make sure that beside the paths to the model simulations and observations, also the path to 
+raw observational data to be cmorized (``RAWOBS``) is present in your configuration file.
+
+3. Store your dataset in the right place
+========================================
+
+The folder ``RAWOBS`` needs the subdirectories ``Tier1``, ``Tier2`` and ``Tier3``. 
+The different tiers describe the different levels of restrictions for downloading 
+and using the observations. The unformatted (raw) observations should then be 
+stored then in the appropriate of these three folders. 
+
+4. Create a cmorizer for the dataset
+========================================================
+
+There are many cmorizing scripts available in `/esmvaltool/cmorizers/obs/
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/>`_
+where solutions to many kinds of format issues with observational data are
+addressed. Most of these scripts are written in NCL at the moment, but more 
+and more examples for Python-based cmorizing scripts become available.
+
+How much cmorizing an observational data set needs is strongly dependent on
+the original NetCDF file and how close the original formatting already is to
+the strict CMOR standard. 
+
+In the following two subsections two cmorizing scripts, one written in python and
+one written in NCL, are explained in more detail.
+
+4.1 Cmorizer script written in python
+*************************************
+
+Find here an example of a cmorizing script, written for the ``MTE`` dataset
+that is available at the MPI for Biogeochemistry in Jena: `cmorize_obs_mte.py
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmorize_obs_mte.py>`_.
+
+All the necessary information about the dataset to write the filename correctly, 
+and which variable is of interest, is stored in a seperate configuration file: `MTE.yml
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/cmor_config/MTE.yml>`_.
+
+The first part of this configuration file defines the filename of the raw
+observations file, the second part defines the common global attributes for 
+the cmorizer output, e.g. information that is needed to piece together the 
+final observations file name in the correct structure (see Section 6). The 
+third part defines the variables that are supposed to be cmorized.
+
+The actual cmorizing script ``cmorize_obs_mte.py`` consists of a header with
+information on where and how to download the data, noting the last access of the
+data webpage, and who has written the cmorizing script. 
```

Comment:
It's not necessary to add the name of the author, `git blame` will do that :grin:

Reply:
That's a remnant of v1, when we used to document the history in the header.
We still do that in the NCL and R routines, but as you say, since we use git we could get rid of that at some point.

Reply:
I took out the last part of the sentence.